src/main/java/com/palantir/gradle/versions/GradleWorkarounds.java

```diff
@@ -124,5 +130,82 @@ static boolean isFailOnVersionConflict(Configuration conf) {
         return conflictResolution == org.gradle.api.internal.artifacts.configurations.ConflictResolution.strict;
     }
 
+    @SuppressWarnings("CyclomaticComplexity")
+    public static void makeEvaluationDependOnSubprojectsToBeEvaluated(Project rootProject) {
+        if (!rootProject.getGradle().getStartParameter().isConfigureOnDemand()
+                || rootProject.getGradle().getStartParameter().isWriteDependencyLocks()
+                // If Gradle is run from somewhere other than the root, the task location gets trickier to translate
+                // into the projects to use; this could be implemented in the future if there's demand
+                || !rootProject.getGradle().getStartParameter().getCurrentDir().equals(rootProject.getRootDir())) {
```

Comment:
So if you run gradle from a subdir, it will still *work*, just configure on demand will not be enabled?

Reply:
Correct.

Reply:
Oh slick, I had no idea gradle supported running within a subdir to limit task breadth! Good flag.